cachi2/core/package_managers/general.py

```diff
@@ -45,7 +46,8 @@ def download_binary_file(
     timeout = get_config().requests_timeout
     try:
         resp = pkg_requests_session.get(
-            url, stream=True, verify=not insecure, auth=auth, timeout=timeout
+            # todo verify needs to be : not insecure
+            url, stream=True, verify=not insecure, auth=auth, timeout=timeout, cert=get_client_certs()
```

Comment:
It isn't implemented yet but it was from thinking about implementing cli options (and needing to get the client cert names from the caller instead of from the env.  I also don't know what 'download_binary_file' is for - it seems to be unused when I fetch RPMs (at least nothing fails when authentication is not configured there).